http/codegen/testdata/result_encode_functions.go

```diff
@@ -625,6 +625,40 @@ func EncodeMethodBodyUserResponse(encoder func(context.Context, http.ResponseWri
 }
 `
 
+var ResultBodyMultipleViewsEncodeCode = `// EncodeMethodBodyMultipleViewResponse returns an encoder for responses
+// returned by the ServiceBodyMultipleView MethodBodyMultipleView endpoint.
+func EncodeMethodBodyMultipleViewResponse(encoder func(context.Context, http.ResponseWriter) goahttp.Encoder) func(context.Context, http.ResponseWriter, interface{}) error {
+	return func(ctx context.Context, w http.ResponseWriter, v interface{}) error {
+		vres := v.(*servicebodymultipleviewviews.Resulttypemultipleviews)
```

Comment:
Wondering why the generated type name isn't properly camelized?

Reply:
That's because I set the result like this `ResultType("ResultTypeMultipleViews", func() {` . So the result type identifier (all lower-case after calling mime.ParseMediaType) is being used as the type name.